Helper/AlgoliaHelper.php

```diff
@@ -757,9 +769,9 @@ public function isValidFloat(string $value) : bool {
 
     /**
      * @param $value
-     * @return float|int
+     * @return mixed
      */
-    protected function castAttribute($value)
+    protected function castAttribute($value): mixed
```

Comment:
That's what I put originally, but while I tested it, it broke the indexing.
The fact is this current @return tag is wrong, the method return basically anything (I had to put `float|int|string|null|array` to make it work again, so I thought that `mixed` would be better.
We can also not specify any return type for this one if you prefer, WDYT ?

Reply:
You know on second look I can see that it might not always return those values. It's an odd bit of code - I say keep it `mixed` then. If anything it calls out a need for a potential refactor.